tests/x11/multi_users_dm.pm

```diff
@@ -89,7 +100,7 @@ sub run {
     script_run "mv /etc/sysconfig/displaymanager.bak /etc/sysconfig/displaymanager";
     assert_script_run "~$username/data/delete_users $users_to_create";
     script_run "clear";
-    restart_x11;
+    $self->restart_x11(0);
```

Comment:
We don't really need to pass 0, as $setnologin will be undef and both are false. But that's up to you.